lib/senkyoshi/models/wikipage.rb

```diff
@@ -18,7 +18,9 @@ def canvas_conversion(course, resources)
 
         # Add page links to page body
         @files.each do |file|
-          if course.files.detect { |f| f.identifier == file.name }
+          if canvas_file = course.files.detect { |f| f.identifier == file.name }
```

Comment:
What if we moved this to the ContentFile canvas conversion method? Then only code in that class would be modifying it's attributes.